tests/run_tests.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
+test_folder=$(dirname "${BASH_SOURCE[0]}")
+root=$(dirname $test_folder)
 
-python -m pip install --upgrade pyright
-python -m pip install -r requirements.txt
+python -m pip install --upgrade -r $test_folder/requirements.txt
 
-
-pyright --pythonversion 3.11 -p pyrighttestconfig.json .
+cd $root
+python -m mypy .
+pyright .
```

Comment:
I don't remember if we discussed preferring using the pyright wrapper on PyPI or using `npx pyright@latest` here.

Reply:
Nope! Just wanted to double-check!
Might be a bit easier for contributors who don't already have node installed.